vernac/record.ml

```diff
@@ -341,6 +341,56 @@ let instantiate_possibly_recursive_type ind u ntypes paramdecls fields =
 
 (* TODO: refactor the declaration part here; this requires some
    surgery as Evarutil.finalize is called too early in the path *)
+let build_named_proj ~primitive ~flags ~poly ~univs ~uinstance ~kind env paramdecls paramargs decl impls fid subst sp_projs nfi ti i indsp mib lifted_fields x rp =
```

Comment:
this line is getting long and should be split

Reply:
done thanks